tools/scripts/mcufla.sh

```diff
@@ -33,17 +33,18 @@
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 USAGE=$(cat <<-END
-Flash maxim microcontrollers using Maxim's own openocd.
-Only use .elf files, won't work with .hex files.
+Flash maxim, stm32 and pico microcontrollers, it downloads specific openocd for Maxim and STM32
+platform, for Pico standard openocd is required.
+Can be used only with .elf files.
 
 Dependencies (Debian):
-	autoconf automake make git libudev-dev libtool libusb-1.0-0-dev pkg-config texinfo
+	autoconf automake make git libudev-dev libtool libusb-1.0-0-dev pkg-config texinfo openocd
```

Comment:
`openocd` is downloaded by this script. I think this list is only meant to include dependencies which have to be installed by the user.